send_message.py

```
@@ -144,24 +165,50 @@ def handle_connection(epic_socket: socket.socket, messages: dict):
+
+                success = True
+                break
+
+            except BrokenPipeError:
+                attempt += 1
+                logger.error(
+                    (
+                        "Failed to send message (probably due to connection "
+                        "reset on Epic side. Attempting to reconnect...)"
+                    )
+                )
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)